lib/crypto-browser.js

```diff
@@ -1,6 +1,11 @@
 /*!
  * Copyright (c) 2019-2023 Digital Bazaar, Inc. All rights reserved.
  */
+// eslint-disable-next-line unicorn/prefer-node-protocol
+import {Buffer} from 'buffer';
+// eslint-disable-next-line no-undef
+window.Buffer = Buffer;
```

Comment:
`Buffer` is a node.js specific API and will likely never be available on the Web platform. However, every instance of a node.js `Buffer` is also an instance of `Uint8Array` which is available on the Web platform. This should be removed and `Uint8Array` should be used wherever a `Buffer` is needed.

Reply:
I tested wrapping signatures in `Uint8Array` a couple days ago and this caused downstream errors while testing in `ecdsa-cryptosuite-2019`. This is why I decided to wrap it in a `Buffer` instead (which is also the signature format for `eddsa-2022-cryptosuite` signatures). I will try to reproduce the error and include a trace here soon.

Reply:
Actually, it seems that the reason for the failure was that I didn't resolve the call to `webcrypto.subtle.sign` with an `await` before passing it into the `Uint8Array`. Fix incoming...